azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -181,6 +182,25 @@ def disable(self):
             self._cgroups_enabled = False
             CGroupsTelemetry.reset()
 
+        def create_slices(self):
+            # Create root slices for agent and agent and extensions for systemd-managed distros.
+            # The hierarchy is as follows:
+            # ├─user.slice
+            # ...
+            # ├─system.slice
+            # ...
+            # └─azure.slice
+            #   └─azure-vmextensions.slice
+
+            # Both methods will log info on success, log warning and emit telemetry on failure.
```

Comment:
let's emit telemetry on success as well so that we can eventually remove them when they are no longer needed